### ISSUE TITLE
Set and then use language for distinguishing normative from non-normative content

### ISSUE THREAD

**Issue by jmbrule** [oasis-tcs/openc2-apsc-stateless-packet-filter#39]
This is a tab comment 1638.  It states "There is no description or use of language to distinguish normative from non-normative text, including appendices. Sections 1.6, 1.7 and 1.8 for instance. But there are other notes, examples, appendices, all of which should be non-normative."

**Comment by dlemire60** [comment]
I went looking at some recently announced committee specifications to see how they dealt with identifying normative / non-normative content, per the TAB comments we received on all three specifications. What I see:
1) The Universal Business Language OASIS Standard appends appropriate section headers with "(Non-normative)"
2) The OSLC Architecture Management committee specs tag non-normative sections only with _This section is non-normative._ immediately after the section header
3) The TOSCA committee spec embeds words about normative or non- into the intro text for each major section (1.0, 2.0), for example "Except for the examples, this section is *normative* and ..."

Chet Ensign indicate a preference for approach #2, which I also like. Based on the examples I've looked at, it seems to be acceptable to identify just the non-normative sections, leaving everything else as implicitly normative. I'm going to create a pull request for the HTTPS spec using this approach.
